infra/cifuzz/fuzz_target.py

```diff
@@ -41,20 +48,23 @@ class FuzzTarget:
     target_name: The name of the fuzz target.
     duration: The length of time in seconds that the target should run.
     target_path: The location of the fuzz target binary.
+    project_name: The name of the relevant OSS-Fuzz project.
   """
 
-  def __init__(self, target_path, duration, out_dir):
+  def __init__(self, target_path, duration, out_dir, project_name=None):
```

Comment:
does it ever make sense for `project_name` to be None? If so, document when this is the case.

Reply:
Added a comment.